packages/golden_toolkit/test/shadows/flutter_test_config.dart

```diff
@@ -0,0 +1,29 @@
+/// ***************************************************
+/// Copyright 2019-2020 eBay Inc.
+///
+/// Use of this source code is governed by a BSD-style
+/// license that can be found in the LICENSE file or at
+/// https://opensource.org/licenses/BSD-3-Clause
+/// ***************************************************
+///
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:golden_toolkit/golden_toolkit.dart';
+
+Future<void> main(FutureOr<void> Function() testMain) async {
+  return GoldenToolkit.runWithConfiguration(
+    () async {
+      await testMain();
+    },
+    config: GoldenToolkitConfiguration(
+      // Currently, goldens are not generated/validated in CI for this repo. We have settled on the goldens for this package
+      // being captured/validated by developers running on MacOSX. We may revisit this in the future if there is a reason to invest
+      // in more sophistication
+      skipGoldenAssertion: () => !Platform.isMacOS,
+      // enableRealShadows can only work when "outside" of a `testGoldens` , for instance here at the root
+      enableRealShadows: true,
+    ),
```

Comment:
in the `shadows` folder, we set `enableRealShadows: true` , and all tests in this folder or subfolder will auto-magically have "nice shadows"